File: official/docs/python/current/shipments/label.py
import os

import easypost


client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))

shipment = client.shipment.retrieve("shp_...")

shipment_with_label = client.shipment.label(shipment.id, file_format="ZPL")

print(shipment_with_label)

File: official/docs/node/current/shipments/label.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const shipment = await client.Shipment.retrieve('shp_...');

  const shipmentWithLabel = await client.Shipment.convertLabelFormat(shipment.id, 'ZPL');

  console.log(shipmentWithLabel);
})();
